### ISSUE TITLE
[CI Failure]:  mi300_1: Multi-Modal Models (Extended Generation 3)

### ISSUE THREAD

**Issue by AndreasKaratzas** [vllm-project/vllm#41854]
### Basic information

- [ ] Flaky test
- [x] Can reproduce locally
- [ ] Caused by external libraries (e.g. bug in `transformers`)

### 🧪 Describe the failing test

```
FAILED models/multimodal/generation/test_common.py::test_single_image_models[aria-test_case127]
FAILED models/multimodal/generation/test_common.py::test_single_image_models[glm4v-test_case134]
FAILED models/multimodal/generation/test_common.py::test_single_image_models[glm4v-test_case135]
FAILED models/multimodal/generation/test_common.py::test_single_image_models[glm4v-test_case136]
FAILED models/multimodal/generation/test_common.py::test_multi_image_models[aria-test_case98]
FAILED models/multimodal/generation/test_common.py::test_multi_image_models[glm_ocr-test_case102]
FAILED models/multimodal/generation/test_common.py::test_multi_image_models[glm_ocr-test_case103]
FAILED models/multimodal/generation/test_common.py::test_multi_image_models[glm_ocr-test_case104]
```

### 📝 History of failing test

- Latest nightly date: 2026-05-06
- Latest build(s): [amd-ci #8223](https://buildkite.com/vllm/amd-ci/builds/8223)
- Latest hardware status: `mi300_1`=fail

**Comment by rasmith** [comment]
working on this

**Comment by rasmith** [comment]
https://github.com/vllm-project/vllm/pull/42126
